.travis.yml

```
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-exporter is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+notifications:
+  email: false
+
+sudo: false
+
+language: python
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    # To allow failures, you need to specify the full environment
+    - env: REQUIREMENTS=devel
+
+cache:
+  - pip
+
+env:
+  - REQUIREMENTS=lowest
+  - REQUIREMENTS=release DEPLOY=true
```

moderate: We have to update the cookiecutter code... but as per https://github.com/inveniosoftware/rfcs/blob/master/rfcs/framework-0016-python2.md we can simplify a bit the travis file.
```suggestion
  - REQUIREMENTS=release
```